enterprise/frontend/src/embedding-sdk/types/collection.ts

```diff
@@ -8,6 +8,54 @@ import type { SdkEntityId } from "./entity-id";
 export type SdkCollectionId = number | "personal" | "root" | SdkEntityId;
 
 /**
- * TODO: replace the internal type with a proper one
+ * The Collection entity
  */
-export type CollectionItem = InternalCollectionItem;
+export type MetabaseCollection = {
+  id: SdkCollectionId;
+  name: string;
+  slug?: string;
+  entity_id?: SdkEntityId;
+  description: string | null;
+  children?: MetabaseCollection[];
+  type?: "instance-analytics" | "trash" | null;
+
+  parent_id?: SdkCollectionId | null;
+  personal_owner_id?: SdkUserId;
+  location: string | null;
+};
+
+/**
+ * The CollectionItem entity
+ */
+export type MetabaseCollectionItem = {
+  id: SdkCollectionId;
+  entity_id?: SdkEntityId;
+  model: string;
+  name: string;
+  description: string | null;
+  archived: boolean;
```

Comment:
If we hide archived items by default in the collection browser, we don't need this